sklearn/preprocessing/imputation.py

```diff
@@ -377,7 +438,7 @@ def transform(self, X):
                 Transformed array.
                 shape (n_samples, n_features_new) when
                 ``add_indicator_features`` is False,
-                shape (n_samples, n_features_new + len(imputed_features_)
+                shape (n_samples, n_features + n_features_new)
```

Comment:
Sorry for the poor choice of words. In case a whole feature is discarded due to all missing values, ``n_features_new`` will be the number of features in the imputer output without indicator_features. In that context, it should be ``n_features_new + len(imputed_features_)``. ``n_features_new`` might not be equal to ``len(imputed_features_)`` in the case of ``axis=0``, since ``imputed_features_`` can discard the features